keyboards/handwired/ndk/ndk16/readme.md

```diff
@@ -0,0 +1,25 @@
+# ndk16
+
+![ndk16](https://github.com/itskevin-zz/ndk-keyboards/assets/7293885/739dfeeb-6cf0-4c67-9100-5f9a37c662bf)
```

Comment:
This image needs to be rehosted on Imgur.

Reply:
Changed url to Imgur